Casks/s/scribus.rb

```diff
@@ -1,19 +1,19 @@
 cask "scribus" do
-  version "1.5.8"
-  sha256 "d5ebc6f104bb57457c68ce651864053040af38f218bad1eef17491db7e6282ef"
+  arch arm: "-arm64"
 
-  url "https://downloads.sourceforge.net/scribus/scribus-devel/#{version}/scribus-#{version}.dmg",
+  version "1.6.1"
+  sha256 arm:   "2de62c9312bc0a9532bf6db6eb479b0cb56b10434a0e6aa47de059d3c881ecc8",
+         intel: "b6f788c30b1ca8f628f3d1851859e1831705520e36b59090e5705b7ebc7fcfeb"
+
+  url "https://downloads.sourceforge.net/scribus/scribus/#{version}/scribus-#{version}#{arch}.dmg",
       verified: "sourceforge.net/scribus/"
   name "Scribus"
   desc "Free and open-source page layout program"
   homepage "https://www.scribus.net/"
 
-  # The stable branch is outdated and supported on deprecated versions of MacOS.
-  # Use the devel branch until the stable branch is updated
-  # https://github.com/Homebrew/homebrew-cask/pull/120289
   livecheck do
-    url "https://www.scribus.net/downloads/"
-    regex(%r{href=.*?scribus[._-]devel/v?(\d+(?:\.\d+)+)}i)
+    url "https://sourceforge.net/projects/scribus/rss?path=/scribus"
+    strategy :sourceforge
```

Comment:
```suggestion
    regex(%r{url=.*?/scribus[._-]v?(\d+(?:\.\d+)+)(?:#{arch})?\.(?:dmg|pkg)}i)
```

livecheck will use the `Sourceforge` strategy for this URL, so `strategy :sourceforge` isn't needed here.

I've also added a regex to this suggestion that will specifically match the files for macOS, so livecheck won't surface versions if there isn't an appropriate file (I periodically see this in other projects).